fp/image-names.js

```diff
@@ -35,15 +36,41 @@ function createImageName(image, types, paramValues) {
           }
         });
         return ret;
+      },
+
+      getShortName: function () {
+        var shortName = getImageName();
+        if (shortName.indexOf("/") > 0) {
+          shortName = shortName.substring(shortName.indexOf("/") + 1);
+        }
+        return shortName;
       }
     };
 
   // ==========================================================================================
 
   function getImageName()
   {
+    var dockerOrg = getDockerOrg();
+    var name = image.name;
+    if (dockerOrg && name.includes("/")) {
+      name = dockerOrg + name.substring(name.indexOf("/"));
+    }
     var registry = image.config.fpConfig('registry') ? image.config.fpConfig('registry') + "/" : "";
-    return registry + image.name + "-" + paramValues.join("-");
+    return registry + name + "-" + paramValues.join("-");
+  }
+
+  function getDockerOrg()
+  {
+    var dockerOrg = undefined;
+    for (type in types) {
+      var param = paramValues[type];
+      var org = image.config.config[types[type]][param].dockerOrg;
```

Comment:
Every parameter which has a meaning for fish-pepper should go below the fish-pepper configuration section. Everything else can be introduced and changed by the user freely. So that kind of dockerOrg should not into this section where the user has the full control of all parameter keys.